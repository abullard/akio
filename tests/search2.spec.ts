import { describe, it, expect } from "vitest";
import { spawnWrapper } from "./test-utils";

describe('search2.spec.ts', () => {
    it('should support monorepo archeology and script execution', async () => {
        const cmd = "pnpm";
        const args = ['akio', 'test:stub'];

        const { stdout } = await spawnWrapper(cmd, args, '5');

        expect(stdout).toMatchSnapshot();
    });
});
