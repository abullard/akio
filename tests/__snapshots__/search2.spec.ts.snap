// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`search2.spec.ts > should run search for test:sub npm scripts, run test:stub 1`] = `
"
> npx akio test\\:stub

[33mpnpm akio[0m
	-----
Found scripts matching: "[32mtest:stub[0m"

1. [35mtest:stub [0m â€” DO NOT REMOVE - stubbed npm script for snapshot tests

[34mRun command number? >[0m 


> node -e "console.log('text')"

text"
`;
